imageio/plugins/tifffile.py

```diff
@@ -529,6 +531,17 @@ def _sanitize_meta(meta):
                     # 1(=NONE) translation to False expected by TiffWriter.save
                     if key == "predictor" and not isinstance(value, bool):
                         ret[key] = value > 1
+                    elif key == "compress" and value != 0:
+                        warnings.warn(
+                            "The use of `compress` is deprecated. Use `compression` and `compressionargs` instead.",
+                            DeprecationWarning,
+                        )
+
+                        if _tifffile.__version__ < "2022":
```

Comment:
Does that mean there's no any version using `compress`?

Reply:
Yes. `compress` was used at some point to facilitate compression and while I didn't check the version of tifffile where this change happened, it has been deprecated for at least one year.